app/uk/gov/hmrc/plasticpackagingtax/registration/connectors/grs/SoleTraderGrsConnector.scala

```diff
@@ -37,7 +37,7 @@ class SoleTraderGrsConnector @Inject() (
     extends GrsConnector[SoleTraderGrsCreateRequest, GrsSoleTraderDetails, SoleTraderDetails](
       httpClient,
       metrics,
-      Some(config.soleTraderJourneyUrl),
+      Some(config.soleTraderJourneyInitUrl),
```

Comment:
I dont think needs to be changed?

Reply:
Actually, we might get away with not changing but the original URL has been deprecated so we would need to change at some point. We might as well go for it now?